frontend/src/core/placeable/parsers/shortCapitalNumberString.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+
+/**
+ * Marks 2-letters-long terms containing a combination of a capital letter and
+ * a number.
```

Comment:
Why 2-letter only? There are longer Numeronyms, e.g. W3C.

Reply:
That will be caught by the `allCapitalsString` placeable.

Reply:
Right!